docs/perf-tests-region-scheduling.md

```
@@ -0,0 +1,74 @@
+RELEASE: scylla-enterprise-perf-regression-predefined-throughput-steps-i8g-tablets - us-east-1
+
+MASTER (3-weeks): scylla-enterprise-perf-regression-latency-650gb-during-rolling-upgrade-i8g-tablets - us-west-2
+RELEASE: scylla-enterprise-perf-regression-latency-650gb-during-rolling-upgrade-i8g-tablets - us-west-2
+
+MASTER (3-weeks): scylla-enterprise-perf-regression-latency-650gb-with-nemesis-i8g-tablets - us-east-2
+RELEASE: scylla-enterprise-perf-regression-latency-650gb-with-nemesis-i8g-tablets - us-east-2
+```
+
+## Available Regions
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)